app/views/admin/applications/show.html.erb

```diff
@@ -1,14 +1,36 @@
 <% content_for :before_content do %>
   <%= render GovukComponent::BackLinkComponent.new(
     text: "Back",
-    href: admin_applications_path
+    href: url_for(:back)
```

Comment:
This page can be reached from one of two indexes, this will take the user back to the correct one.